backend/qemu.pm

```diff
@@ -832,7 +832,7 @@ sub start_qemu {
             sp('k', $vars->{VNCKB}) if $vars->{VNCKB};
         }
 
-        if ($vars->{VIRTIO_CONSOLE}) {
+        if ($vars->{VIRTIO_CONSOLE} ne 0) {
```

Comment:
I think it would be better to give VIRTIO_CONSOLE a default value of 1 near the top of this function. This means VIRTIO_CONSOLE will always appear in the vars.json when the feature is being used. Also I think most people would expect that if the variable is undefined then the feature will be disabled.

Reply:
In the distro or in the backend? The backend should definitely return an error if someone requests virtio-console when it is disabled.

Reply:
I meant creating test in `t/10-terminal.t` in this project (to increase keep Codecov happy).

Reply:
I guess so. I am not sure codecov will ever be happy though :-)

Reply:
I added testing `VIRTIO_CONSOLE=0`, but even this increased testing code didn't make Codecov happy. Any objections, or can this be included?

Reply:
I see your test so I'd say forget about codecov. Measuring the coverage for forks just doesn't work correctly.